oar/core/shipment.py

```
@@ -1526,7 +1526,7 @@ def check_cve_tracker_bug(self):
 
         logger.debug(f"elliott cmd {cmd}")
 
-        p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=get_elliott_env())
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)